app/models/Customers.py

```diff
@@ -0,0 +1,34 @@
+from flask import current_app
+from app import db 
+from sqlalchemy.orm import relationship
+from datetime import datetime
+
+
+
+class Customer(db.Model):
+    __tablename__ = "customers"
+    customer_id = db.Column(db.Integer, primary_key=True)
+    customer_name = db.Column(db.String(64))
+    customer_zip = db.Column(db.String(5))
+    customer_phone = db.Column(db.String(14))
+    register_at = db.Column(db.DateTime)
+    videos_checked_out_count = db.Column(db.Integer)
```

Comment:
Instead of this being a column you can use the rentals to compute this.  

To do that it's helpful to add a relationship:
```suggestion
    rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```